spec/motion-spec/bacon_spec.rb

```diff
@@ -337,41 +334,51 @@ def the_meaning_of_life
     42
   end
 
-  it "should be accessible in a test" do
+  it "is accessible in a test" do
     the_meaning_of_life.should.eq 42
   end
 
   describe "when in a sibling context" do
-    it "should be accessible in a test" do
+    it "is accessible in a test" do
       the_meaning_of_life.should.eq 42
     end
   end
 end
 
 describe 'describe arguments' do
+  # These specs are testing describe and each time describe gets called a new
+  # context is popped onto the stack. This leads to some seemingly random
+  # empty output at the end of the specs so let's just pop any newly added
+  # contexts off that stack.
+  before { @before_context_count = MotionSpec.instance_variable_get('@contexts').count }
+  after do
+    @contexts = MotionSpec.instance_variable_get('@contexts')
+    (@contexts.count - @before_context_count).times { @contexts.pop }
+  end
+
   def check(ctx, name)
-    ctx.class.ancestors.should.include Motion::Spec::Context
+    ctx.class.ancestors.should.include MotionSpec::Context
     ctx.instance_variable_get('@name').should.eq name
   end
 
-  it 'should work with string' do
+  it 'works with string' do
     check(Kernel.send(:describe, 'string') {}, 'string')
   end
 
-  it 'should work with symbols' do
-    check(Kernel.send(:describe, :behaviour) {}, 'behaviour')
+  it 'works with symbols' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.